docs/dev/models/dsettlement.rst

```diff
@@ -16,6 +16,32 @@ Tutorial
 --------
 You can find the D-Settlement tutorial here: :doc:`../../community/tutorial_dsettlement`
 
+Fit for settlement plate
+--------
+With a fit calculation, the match between predicted and measured settlements can be improved by automatic scaling of soil properties.
+A close fit will improve the continued prediction of final and residual settlements.
+To perform a fit for settlement plate calculation, the following settings are required:
+
+.. code-block:: python
+
+    # In the model settings, the fit option must be activated:
+    dm.set_model(is_fit_for_settlement_plate=True)
```

Comment:
I don't think this works. I tried it in test_run_fit. Please show me.

Reply:
it doesn't work idd, line is removed